index.js

```diff
@@ -33,6 +33,14 @@ module.exports = function inspect_ (obj, opts, depth, seen) {
     if (typeof obj === 'string') {
         return inspectString(obj);
     }
+    if (typeof obj === 'number') {
+      if (Object.is) {
```

Comment:
this should still pass when `Object.is` is not present - it'd probably be simpler to do:
```js
if (obj === 0) {
  return Infinity / 0 > 0 ? '0' : '-0';
}
```